admin/views/accessibility-statement.ejs

```diff
@@ -0,0 +1,176 @@
+<div class="grid-row">
+    <div class="column-two-thirds">
+        <%- partial('partials/_breadcrumbs', {breadcrumbs: breadcrumbs}) %>
+    </div>
+    <div class="column-one-third">
+        <%- partial('partials/_name_and_sign_out') %>
+    </div>
+</div>
+
+<div class="grid-row">
+    <div class="column-two-thirds">
+        <h1 class="heading-xlarge">Accessibility statement</h1>
+
+        <%- partial('partials/_session_expiry') %>
+        
+        <p>
+            We're constantly working to make the multiplication tables check (MTC) accessible and user-friendly for all
+            pupils, regardless of physical, cognitive or emotional impairment.
+        </p>
+
+        <p>
+            You can read a <a target="_blank" rel="noopener noreferrer" href="https://accessibility.blog.gov.uk/">
+            blog about accessibility</a> <img alt="" src="/images/icon-link.png"/> on GOV.UK. 
+        </p>
+
+        <h3 class="heading-medium">Our commitment to accessibility</h3>
+
+        <p>
+            The Standards and Testing Agency is committed to making websites accessible, in accordance with The Public 
+            Sector Bodies (Websites and Mobile Applications) Accessibility Regulations 2018.
+        </p>
+
+        <p>This accessibility statement applies to the MTC:</p>
+
+        <ul class="list list-bullet">
+            <li>pupil site</li>
+            <li>teacher site</li>
+        </ul>
+
+        <h3 class="heading-medium">Conformance status</h3>
+
+        <p>
+            We seek to ensure that all aspects of the service meet the
+            <a target="_blank" rel="noopener noreferrer" href="https://www.w3.org/WAI/standards-guidelines/wcag/">
+            Web Content Accessibility Guidelines (WCAG)</a> <img alt="" src="/images/icon-link.png"/>  
+            2.1 at Level AA. The MTC is partially conformant with WCAG 2.1 level AA.
+        </p>
+
+        <p>Partially conformant means that some parts of the content do not fully conform to the accessibility standard.</p>
+
+        <h3 class="heading-medium">Pupil site</h3>
+
+        <p>The MTC does not support third-party screen readers due to compatibility issues with the timed element of the check.</p>
+        
+        <p>
+            To address this, MTC provides a number of access arrangements that can be applied to the pupil check. The following 
+            table shows which access arrangements are compatible with each device.
+        </p>
+
+        <table class="table-font-xsmall">
```

Comment:
NonBlocker: It seems likely that putting text in a table element makes it harder to hear when a screen reader is used.